python_modules/dagster/dagster/core/execution/execute_in_process.py

```diff
@@ -36,6 +37,7 @@ def core_execute_in_process(
     raise_on_error: bool,
     run_tags: Optional[Dict[str, Any]] = None,
     run_id: Optional[str] = None,
+    asset_selection: Optional[FrozenSet[AssetKey]] = None,
```

Comment:
hmm - seems odd here to have asset selection but not op selection, why is that?

Reply:
When jobs are executed via `execute_in_process`, op selection is not passed to the `PipelineRun` object. I modified this to pass asset_selection to `PipelineRun`, but op selection still does not exist